src/status_im/ui/screens/chat/message/message_old.cljs

```diff
@@ -86,28 +86,51 @@
          :accessibility-label :message-timestamp}
         timestamp-str]])))
 
+
 (defview quoted-message
-  [_ {:keys [from parsed-text image]} outgoing current-public-key public? pinned]
+  [_ {:keys [from parsed-text image audio sticker id] :as message} outgoing current-public-key public? pinned chat-id]
   (letsubs [contact-name [:contacts/contact-name-by-identity from]]
-    [react/view {:style (style/quoted-message-container (and outgoing (not pinned)))}
-     [react/view {:style style/quoted-message-author-container}
-      [chat.utils/format-reply-author
-       from
-       contact-name
-       current-public-key
-       (partial style/quoted-message-author (and outgoing (not pinned)))
-       (and outgoing (not pinned))]]
-     (if (and image
-              ;; Disabling images for public-chats
-              (not public?))
-       [fast-image/fast-image {:style  {:width            56
-                                        :height           56
-                                        :background-color :black
-                                        :border-radius    4}
-                               :source {:uri image}}]
-       [react/text {:style           (style/quoted-message-text (and outgoing (not pinned)))
+           [react/view {:style (style/quoted-message-container (and outgoing (not pinned)))}
+            [react/view {:style style/quoted-message-author-container}
+             [chat.utils/format-reply-author
+              from
+              contact-name
+              current-public-key
+              (partial style/quoted-message-author (and outgoing (not pinned)))
+              (and outgoing (not pinned))]]
+            (if (and image
+                     ;; Disabling images for public-chats
+                     (not public?))
+              [fast-image/fast-image
+               {:style  {:width            56
+                         :height           56
+                         :background-color :black
+                         :border-radius    4}
+                :source {:uri image}}]
+
+              (if (and audio
```

Comment:
Better to use [cond](https://clojuredocs.org/clojure.core/cond), instead of multiple `if`